### PR TITLE
[BEAM-5068] switch mobile gaming input file to a public storage

### PR DESCRIPTION
**Pull request by yifanzou**
Switch the mobile-gaming input date "5000_gaming_data.csv" to a public storage:
1. Allows contributors run this test on their local machine;
2. No worries about the life time in the gcs under the apache-beam-testing
+R @pabloem 
cc: @alanmyrvold @boyuanzz  

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by yifanzou** [comment]
Run Seed Job

**Comment by yifanzou** [comment]
Run Dataflow PostRelease

**Comment by yifanzou** [comment]
Run Seed Job

**Comment by yifanzou** [comment]
Run Dataflow PostRelease

**Comment by yifanzou** [comment]
Run Seed Job

**Comment by yifanzou** [comment]
Run Dataflow PostRelease

**Comment by yifanzou** [comment]
Run Seed Job

**Comment by yifanzou** [comment]
Run Dataflow PostRelease

**Comment by yifanzou** [comment]
Run Seed Job

**Comment by yifanzou** [comment]
Run Dataflow PostRelease

File: .test-infra/jenkins/job_PostRelease_NightlySnapshot.groovy [hunk 1]
```diff
@@ -23,9 +23,6 @@ import CommonJobProperties as commonJobProperties
 job('beam_PostRelease_NightlySnapshot') {
   description('Runs post release verification of the nightly snapshot.')
 
-  // Execute concurrent builds if necessary.
-  concurrentBuild()
```
Comment (pabloem) [inline .test-infra/jenkins/job_PostRelease_NightlySnapshot.groovy]: Why are you removing it now? And why did we add it before? (I think it makes sense to remove, just curious what made you do it).

**Comment by pabloem** [comment]
LGTM. I'll merge now, but I've left one comment. Can you answer that?

File: .test-infra/jenkins/job_PostRelease_NightlySnapshot.groovy (hunk 1 above)
Comment (yifanzou) [reply .test-infra/jenkins/job_PostRelease_NightlySnapshot.groovy]: The tests do operations on gcp resources such as bigquery tables and verify results there. Stop concurrent building will prevent multiple tests working on the same table that cause conflicts and failures.

Comment (pabloem) [reply .test-infra/jenkins/job_PostRelease_NightlySnapshot.groovy]: Ah that makes sense. Thanks.

**Pull request merged**